src/Microsoft.Health.Dicom.Api/Extensions/WorkitemResponseStatusExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Net;
+using Microsoft.Health.Dicom.Core.Messages.WorkitemMessages;
+
+namespace Microsoft.Health.Dicom.Api.Extensions
+{
+    public static class WorkitemResponseStatusExtensions
+    {
+        private static readonly IReadOnlyDictionary<WorkitemResponseStatus, HttpStatusCode> StoreResponseStatusToHttpStatusCodeMapping =
+            new Dictionary<WorkitemResponseStatus, HttpStatusCode>()
+            {
+                { WorkitemResponseStatus.None, HttpStatusCode.NoContent },
+                { WorkitemResponseStatus.Success, HttpStatusCode.OK },
+                { WorkitemResponseStatus.Failure, HttpStatusCode.Conflict },
```

Comment:
This needs to be revisited

Reply:
This requires revisiting the Store handler response codes as well. We should consider addressing this in a separate PR.